main.js

```diff
@@ -0,0 +1,17 @@
+var $img = document.querySelector('img');
+
+var car = {
+  direction: ''
+};
+
+function updateDir(event) {
+  if (event.key === 'ArrowRight' ||
+      event.key === 'ArrowDown' ||
+      event.key === 'ArrowLeft' ||
+      event.key === 'ArrowUp') {
+    car.direction = event.key;
+  }
+  $img.setAttribute('class', car.direction);
```

Comment:
move the setAttribute method call into the curly braces so itll only update the direction when the correct key is pressed and not every time a user presses a key.